nautobot/docs/user-guide/platform-functionality/jobs/job-scheduling-and-approvals.md

```diff
@@ -59,14 +59,14 @@ The queue of jobs that need approval can be found under `Approvals > Approval Da
 
 ### Approval via the API
 
-Approvals can also be given via the REST API. The endpoints to approve, deny, and dry run a scheduled job are found on the scheduled job endpoint under `approve`, `deny`, and `dry-run`, respectively.
+Approvals can also be given via the REST API. The endpoints to approve, deny, and dry run a scheduled job are found on the scheduled job endpoint under `approve`, `deny` respectively.
+You may also include a comment in the request data when approving or denying a workflow.
```

Comment:
Let's make sure to document the `comment` and `active-workflows` endpoints as well.